### DISCUSSION TITLE
Billing Issue with downgraded pro plan trial

### DISCUSSION THREAD

**Discussion by handaggle** [vercel/community#7173] [Help] [answered]
### Summary

Hello Vercel Team,

I hope this message finds you well.

I recently upgraded my account from the Hobby plan to the Pro plan trial to use some Pro plan features (specifically, building a project in the team repository). During this time, I registered a credit card for billing purposes. After testing the Pro plan features, I decided to downgrade back to the Hobby plan. My current account status reflects that I am on the Hobby plan.

However, I am still receiving billing alerts for the registered card. No payment has been made due to insufficient funds on the card, but I am concerned about the ongoing alerts.

Could you please clarify what might be causing this issue? I want to ensure that my account remains on the Hobby plan and that I do not incur any charges.

Thank you for your assistance.

### Steps to Reproduce

<img width="1080" alt="스크린샷 2024-06-12 오후 11 55 50" src="https://github.com/vercel/community/assets/169332560/5535f31d-8369-4be2-90ce-e08006f4028c">

**Comment by amyegan** [comment, marked as answer]
Hey @handaggle. If you added a payment method to upgrade to a Pro plan but your card had insufficient funds, then you may still be getting notifications about a past unpaid invoice.

If you believe there was an error with your invoice, please [submit an Invoice Enquiry case to our Billing team](https://vercel.com/help#issues). They'll be able to dig into your account details to help get it sorted.

**Reply by handaggle** [reply]
Hi @amyegan 
Thank you for your help with this issue.

If I upgrade to use the pro plan trial and then cancel it, will I still be charged for the initial upgrade? I upgraded on Friday the 15th, so I understood that I would not be charged because the trial period had not ended.

If there is anything I misunderstood, I would really appreciate it if you could let me know.

[a comment by an automated account is not shown]
